contrib/native/client/src/include/drill/userProperties.hpp

```diff
@@ -28,6 +28,20 @@ class DECLSPEC_DRILL_CLIENT DrillUserProperties{
         static const std::map<std::string, uint32_t> USER_PROPERTIES;
 
         DrillUserProperties(){};
+        
+        /// @brief Insert or update property value associate with the property key if the value is 
+        /// empty or key is undefined. This function is useful for setting default property.
+        /// 
+        /// @param in_propName              The property name.
+        /// @param in_propValue             The property value.
+        void SetDefaultProperty(const std::string& in_propName, const std::string& in_propValue)
```

Comment:
Follow naming convention setDefaultProperty instead of SetDefaultProperty

Reply:
gotcha thanks!